### PR TITLE
fix wrong typeface set for bottom steps text

### PR DESCRIPTION
**Pull request by luongvo**
Before
image

After
image

### CODE REVIEW

**Comment by shuhart** [comment]
Going to close this one.

**Comment by luongvo** [comment]
@shuhart any reason why did you close this PR please? 🤔

**Comment by shuhart** [comment]
Thought it was outdated. Why did you ignored the comment?

File: sample/src/main/java/com/shuhart/stepview/sample/examples/customise/CustomiseActivity.java [hunk 1]
```diff
@@ -38,6 +39,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
     }
 
     private void setupStepView() {
+        stepView.getState().typeface(Typeface.createFromAsset(getAssets(), "font/LemonSansRegular.otf")).commit();
```
[the remark this answers is not in the document]
Comment (luongvo) [reply sample/src/main/java/com/shuhart/stepview/sample/examples/customise/CustomiseActivity.java]: @shuhart it's in sample project, no need :)

**Comment by luongvo** [comment]
@shuhart sr but I did not see your review before closing, anw, thanks for merging 👍

**Pull request merged**